test/karma/package-lock.json

```diff
@@ -39,47 +38,48 @@
       }
     },
     "node_modules/@babel/code-frame": {
-      "version": "7.21.4",
```

Comment:
By chance did an `npm i` get run in this directory? If I uninstall typescript on my machine locally, I don't see these changes.

Reply:
Could it be `karma-typescript` that's keeping it around? They list TS as a peer dep, which I believe should be getting installed by default (and is probably why TS 5.0.4 is still hanging around).

Reply:
I think so yeah, so one solution could be to explicitly install typescript in `test/karma` and use the same version as in our root one

Reply:
In that case, it may be worth re-enabling renovate for this directory. I promised that PR yesterday and failed to deliver 😆 - thought's on us re-enabling that and see how that works out? That way, renovate should bump both TS versions (here and in the root of the project) at the same time

Reply:
https://github.com/ionic-team/stencil/pull/4816 created

Reply:
I think it makes sense!

Reply:
https://github.com/ionic-team/stencil/pull/4820 created to keep things in line